index.html

```diff
@@ -69,5 +69,181 @@ <h1>Hey There. I’m Ronald.</h1>
+        <h2 class="cardheader">Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button>See project</button>
+      </div>
+      <div class="card2">
+        <div class="textitems">
+          <h2>Profesional Art Printing Data</h2>
```

Comment:
## Styling ✨✨

- [ ] Kindly, it would be better to replicate the given ```font-colors, font-size, and the spacing``` of your elements as given by the Figma design 😎😎

   Replicating the font `colors, sizes, and alignments` from the Figma design will help ensure that the final product looks consistent and polished.  😎😎

### You can check the comparison below 👇 👇 
![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/102757126/4f10884d-5a48-4414-8963-28d8874a6bb5)